webcompat/static/js/lib/wizard/step1.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import notify from "./notify.js";
+
+const urlField = $("#url");
+const nextStepButton = $(".next-step.step-1");
+
+//@todo remove this
```

Comment:
(Why do you need to remove this, and the other `@todo`s?)

Reply:
So I haven't moved any logic for disabling/enabling/validation yet and I'll revisit this in the next issues. Right now it's enabling the next button for easier steps navigation testing

Reply:
Got it!